eox_tagging/api/v1/viewset.py

```diff
@@ -207,7 +208,7 @@ class TagViewSet(viewsets.ModelViewSet):
     """Viewset for listing and creating Tags."""
 
     serializer_class = TagSerializer
-    authentication_classes = (BearerAuthentication, SessionAuthentication)
+    authentication_classes = (BearerAuthentication, SessionAuthentication, JwtAuthentication,)
```

Comment:
@bra-i-am unless we have a good reason to enforce immutability for the `authentication_classes` here, I think it's best to stick with lists as we did in the other plugins: 
https://github.com/eduNEXT/eox-manage/pull/25/files#diff-80eed3c6af56abe3c9be25ddfededbc269a567eaa8435f015f5104dc6bdaec1dR46
https://github.com/eduNEXT/eox-support/pull/17/files#diff-c21fbddde5a151dabbde2f7f31d52fe9949c97871c7ca36189918a6c2e0004f8R85